### PR TITLE
simple readme

### PR DESCRIPTION
**Pull request by FredTheDino**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Sylt-lang
+-----------
+Sylt is a statically-checked dynamically typed, reference counted programming
+language made for GameJams.
+
+
+## Why does this exist? Why not use X-lang?
```
Comment (sornas) [inline README.md:7]:
```suggestion
## Why does this exist? Why use this instead of language X?
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+Pfft! Why not?
+    
+
+## Getting started
+Sylt is written entirely in Rust. Pointing to this Git-repo 
+in your Cargo.toml is the easiet way. Alternatively you can
```
Comment (sornas) [inline README.md:13]:
```suggestion
in your Cargo.toml is the easiest way. Alternatively you can
```

File: README.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+build the interpreter and run your own programs from there.
```
Comment (sornas) [inline README.md:14]: Maybe some instructions on how to build? Even if it isn't more than "1. git clone 2. cargo build". Also, right now this paragraph contains two ideas.

File: README.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+
+## Endgame
```
Comment (sornas) [inline README.md:17]:
```suggestion
## Long-term goals
```

File: README.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
+# Sylt-lang
+-----------
+Sylt is a statically-checked dynamically typed, reference counted programming
```
Comment (sornas) [inline README.md:3]:
```suggestion
Sylt is a statically checked and dynamically typed reference counted programming
```

File: README.md [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+language made for GameJams.
```
Comment (sornas) [inline README.md:4]:
```suggestion
language made for game jams.
```

Comment (sornas) [reply README.md:4]: LiU Game Jam is a title, but a game jam is just a normal thing:tm:.

File: README.md (hunk 4 above)
Comment (FredTheDino) [reply README.md:17]: Don't agree, we'll have to discuss this.

**Pull request merged**